test/e2e/atlas/atlas_e2e_test_generator_test.go

```diff
@@ -283,7 +282,7 @@ func deleteOrgTeams(t *testing.T, cliPath string) {
 	var teams atlasv2.PaginatedTeam
 	require.NoError(t, json.Unmarshal(resp, &teams), string(resp))
 	for _, team := range teams.GetResults() {
-		assert.NoError(t, deleteTeam(team.GetId()))
+		require.NoError(t, deleteTeam(team.GetId()))
```

Comment:
can you revert this please, we want to try all and not fail on the first error

Reply:
The changes in this PR related to `require` and `assert` were due to the linter (testifylint) errors:
```
require-error: for error assertions use require (testifylint)
```
https://github.com/Antonboom/testifylint?tab=readme-ov-file#require-error
The pre-commit hook prevents me from pushing without fixing these instances. I'm not sure why this isn't an issue in all branches

Reply:
Turns out I somehow had the incorrect golangci-lint version. All these changes have been reverted in [0d3326d](https://github.com/mongodb/mongodb-atlas-cli/pull/2600/commits/0d3326d98c808cbe9b3d3497d5e6176079523b5a)